pyface/font_dialog.py

```diff
@@ -0,0 +1,40 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" The implementation of a dialog that allows the user to choose a font.
+"""
+
+from .constant import OK
+from .toolkit import toolkit_object
+
+FontDialog = toolkit_object("font_dialog:FontDialog")
+
+
+def get_font(parent, font):
```

Comment:
Shouldn't `parent` be passed to the actual dialog?

Another thing, I might be using this function incorrectly, but simply calling it with `parent=None` and any font, and then clicking on either of the dialog buttons freezes the application with qt. No dialog window is brought up at all if using wx.

And finally, this function isn't tested at all, so might be good to add some tests.

Reply:
Qt dialog now works fine and wx dialog is brought up. But now it's the wx dialog that freezes for quite some time after a button is clicked (either OK, Cancel). I'm guessing that it's related to the closing of the dialog window.